src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -851,7 +852,9 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
       }
       person.deliveryAddress = { ...this.inProgressDeliveryAddress }
     }
-    if (this.isBenBcCccUlcCorrectionFiling) {
+    // Note: For corrections if the appointmentDate isn't included - you may run into some issues where adding a new
+    // director as it wont show up in the parties or directors call.
+    if (this.isBenBcCccUlcCorrectionFiling || this.isCoopCorrectionFiling) {
```

Comment:
I could change this to `this.isCorrection && !this.isFirm`

not sure if that's possible

Reply:
That's possible but please don't use negative logic here -- we want to INCLUDE the desired types, not EXCLUDE certain types as this will fail when we add other types.

Reply:
Probably best to just leave it how it is I think

Reply:
Agreed.